src/tasks/start-task.ts

```diff
@@ -6,15 +6,14 @@ import { BuidlerRuntimeEnvironment } from '@nomiclabs/buidler/types'
 import { aragenMnemonic, aragenAccounts } from '~/src/params'
 import { AragonConfig } from '~/src/types'
 import { logMain } from '~/src/ui/logger'
-import { getAppId } from '~/src/utils/appName'
-import { getAppName, getAppEnsName } from '~/src/utils/arappUtils'
+import { getAppId, getAppNameParts } from '~/src/utils/appName'
 import { readJson, pathExists } from '~/src/utils/fsUtils'
 import onExit from '~/src/utils/onExit'
-import { validateEnsName } from '~/src/utils/validateEnsName'
 import { startBackend } from './start/start-backend'
 import { startClient } from './start/start-client'
 import { startFrontend } from './start/start-frontend'
 import { TASK_START } from './task-names'
+import { readArapp, parseAppName } from '../utils/arappUtils'
```

Comment:
Nitpick: In our current convention this should be `import { readArapp, parseAppName } from '~/src/utils/arappUtils' and place on line 10 🤓